src/app/modules/products/pages/product/product.page.html

```diff
@@ -1 +1,8 @@
-<app-product-list></app-product-list>
+<div class="product-page-container">
+    <div class="product-page-featured-items-container">
+        <div class="product-page-featured-items-card-container" *ngFor="let product of featuredProducts$ | async">
+            <app-featured-product-card [product]="product"></app-featured-product-card>
+        </div>
+    </div>
+    <app-product-list [products$]=products$></app-product-list>
```

Comment:
there is no need to pass the full observable here, we can extract the array before passing it to the list component.

Observables should be defined in the same component where the data is fetched. Otherwise, if we use @input, we can safely assume that the data is present by the time we reach that point in the template. You can also use the async pipe when passing data:

[products]="products$ | async"

Reply:
all good